dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/controller/UsersController.java

```diff
@@ -111,6 +111,10 @@ public Result createUser(@Parameter(hidden = true) @RequestAttribute(value = Con
                              @RequestParam(value = "email") String email,
                              @RequestParam(value = "phone", required = false) String phone,
                              @RequestParam(value = "state", required = false) int state) throws Exception {
+        Result verifyRet = usersService.verifyUserName(userName);
+        if (verifyRet.getCode() != Status.SUCCESS.getCode()) {
+            return verifyRet;
+        }
```

Comment:
Can we also add this check in update api?

Reply:
The update API already does this check.

https://github.com/apache/dolphinscheduler/blob/399d0d0ce9be29724988dcbd9a7dc5fbfab44de0/dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/UsersServiceImpl.java#L408-L432